pysat/_files.py

```diff
@@ -526,35 +810,44 @@ def from_os(cls, data_path=None, format_str=None,
         """
 
         if data_path is None:
-            raise ValueError("Must supply instrument directory path " +
-                             "(dir_path)")
-
-        # parse format string to figure out the search string to use
-        # to identify files in the filesystem
-        # different option required if filename is delimited
-        if delimiter is None:
-            wildcard = False
-        else:
-            wildcard = True
-        search_dict = construct_searchstring_from_format(format_str,
-                                                         wildcard=wildcard)
+            raise ValueError(" ".join(("Must supply instrument directory path",
+                                       "(dir_path)")))
+
+        # Parse format string to figure out which search string should be used
+        # to identify files in the filesystem. Different option required if
+        # filename is delimited
+        wildcard = False if delimiter is None else True
+        search_dict = futils.construct_searchstring_from_format(
+            format_str, wildcard=wildcard)
         search_str = search_dict['search_string']
-        # perform local file search
-        files = search_local_system_formatted_filename(data_path, search_str)
-        # we have a list of files, now we need to extract the information
-        # pull of data from the areas identified by format_str
+
+        # Perform the local file search
+        files = futils.search_local_system_formatted_filename(data_path,
+                                                              search_str)
+
+        # Use the file list to extract the information. Pull data from the
+        # areas identified by format_str
         if delimiter is None:
-            stored = parse_fixed_width_filenames(files, format_str)
+            stored = futils.parse_fixed_width_filenames(files, format_str)
         else:
-            stored = parse_delimited_filenames(files, format_str, delimiter)
-        # process the parsed filenames and return a properly formatted Series
-        return process_parsed_filenames(stored, two_digit_year_break)
+            stored = futils.parse_delimited_filenames(files, format_str,
+                                                      delimiter)
+
+        # Process the parsed filenames and return a properly formatted Series
+        return futils.process_parsed_filenames(stored, two_digit_year_break)
+
+# --------------------------------------------------------
+#   Utilities supporting the Files Object (all deprecated)
 
 
 def process_parsed_filenames(stored, two_digit_year_break=None):
     """Accepts dict with data parsed from filenames and creates
```

Comment:
Remove this and other 4.0 deprecated files.  We'll follow the current strategy and deprecate them here, with warnings in 2.3.0.

Reply:
I have this and some other changes over in #704.

Reply:
Not quite sure why you're doing that there and not just committing to this branch directly now that you know what's what, but as long as it gets addressed, keep swimming forward 🐟

Reply:
Many changes were already in #704 from getting myself situated. I did remove the deprecated functions after your comment, which could've gone here. The other pull needed to get merged in regardless (after review of this pull after your pull was merged) so for my own sanity I decided to keep my changes to one branch at a time.